### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by Waldleufer**
I created a first draft of an issue template. Just so that all Bug reports have a semi-same style and are easier to read.

### CODE REVIEW

**Comment by Waldleufer** [comment]
Ich habe die merge request auf den Master gemacht, weil Automation und Issue Templates auf dem ganzen Projekt arbeiten und deshalb auf dem master liegen müssen um zu funktionieren.

File: .github/workflows/bugAutomation.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  assign_one_project:
+    runs-on: ubuntu-latest
+    name: Assign to Bug Tracker
+    steps:
+    - name: Assign issues with `bug` label to project 3
+      uses: srggrs/assign-one-project-github-action@1.2.1
+      if: |
+        contains(github.event.issue.labels.*.name, 'bug')
+      with:
+        project: 'https://github.com/Waldleufer/TemplateRepository/projects/3'
```
Comment (Waldleufer) [inline .github/workflows/bugAutomation.yml:21]:
```suggestion
        project: 'https://github.com/RealTelefonmast/OfficialTiberiumRim/projects/3'
```

Das muss natürlich so lauten 😄

File: .github/ISSUE_TEMPLATE/bug-report--tiberiumrim-mod-.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+### **Describe the bug**
+  <!--- This should be the last thing you fill out. Start with the other steps first and come back here after that! -->
+A clear and concise description of what the bug is.
+
+### **Describe your game setup**
+  <!--- This can help us a lot! Please pick/replace whatever you see fit most! -->
+1. - [ ] I added TiberiumRim to an older, already created colony 
+   - [ ] I installed TiberiumRim and then created a fresh colony
+2. - [ ] Runnnig **only TiberiumRim Mod (and HugsLib + Harmony)**
```
Comment (RealTelefonmast) [inline .github/ISSUE_TEMPLATE/bug-report--tiberiumrim-mod-.md]: Hugslib is not necessary, only Harmony as a dependency.
Hugslib is useful for log-sharing though, should be recommended separetly.

File: .github/ISSUE_TEMPLATE/bug-report--tiberiumrim-mod-.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+### **Describe the bug**
+  <!--- This should be the last thing you fill out. Start with the other steps first and come back here after that! -->
+A clear and concise description of what the bug is.
+
+### **Describe your game setup**
+  <!--- This can help us a lot! Please pick/replace whatever you see fit most! -->
+1. - I installed TiberiumRim and then created a fresh colony <!--- OR -->
+   - I added TiberiumRim to an older, already created colony 
+2. - Runnnig **only TiberiumRim Mod (and HugsLib + Harmony)** <!--- OR -->
```
Comment (Waldleufer) [inline .github/ISSUE_TEMPLATE/bug-report--tiberiumrim-mod-.md:18]:
```suggestion
2. - Running **only TiberiumRim Mod and Harmony (and possibly HugsLib to make sharing logs easier)** <!--- OR -->
```

Is that better?

**Review by RealTelefonmast — APPROVED** [review]
While some things might be adjusted (ie wording) this is good for now.

File: .github/ISSUE_TEMPLATE/bug-report--tiberiumrim-mod-.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+   - Running other Mods besides TiberimRim, HugsLib and Harmony  <!--- No need to put the other mods here, we can see this in from the ErrorLog -->
+3. Playing with the following **DLCs enabled** <!--- Please delete what doesn't apply -->
+   - Rimworld - **Royalty**
+   - Rimworld - **Ideology**
+
+
+### **To Reproduce**
+Steps to reproduce the behaviour:
```
[the remark this answers is not in the document]
Comment (Waldleufer) [reply .github/ISSUE_TEMPLATE/bug-report--tiberiumrim-mod-.md:26]: That was what the default template suggested, to be honest ;)

**Pull request merged**